bouncer/util.py

```diff
@@ -133,6 +137,7 @@ def parse_document(document):
         "show_metadata": show_metadata,
         "quote": _escape_quotes(quote),
         "text": _escape_quotes(text),
+        "has_media_time": has_media_time,
     }
```

Comment:
The `parse_document` function could be changed to more modern Python at some point and return a data class instead of a dict.